.github/.wordlist.txt

```diff
@@ -143,6 +143,8 @@ bluez
 BooleanState
 bootable
 Bootloader
+BootReason
```

Comment:
Lets use backticks to escape these (generally any command line) since these are not real words.